tests/environment.py

```diff
@@ -265,7 +273,8 @@ class RadioDocker(Radio):
     def __init__(self, agent: ALEntityDocker, iface_name: str):
         self.iface_name = iface_name
         ip_output = agent.command("ip", "-o",  "link", "list", "dev", self.iface_name).decode()
-        mac = re.search(r"link/ether (([0-9a-fA-F]{2}:){5}[0-9a-fA-F]{2})", ip_output).group(1)
+        mac = re.search(r"link/ether (([0-9a-fA-F]{2}:){5}[0-9a-fA-F]{2})",
+                        ip_output).group(1)
```

Comment:
Are this one and the 2 changes above whitespace changes only? 
If they are they should be done in separate commits, otherwise it forces reviewers to play the [Spot the difference game](https://en.wikipedia.org/wiki/Spot_the_difference).